tests/test_crawler.py

```diff
@@ -11,7 +11,10 @@
 from scrapy.utils.log import configure_logging, get_scrapy_root_handler
 from scrapy.utils.spider import DefaultSpider
 from scrapy.utils.misc import load_object
+from scrapy.utils.test import get_crawler
 from scrapy.extensions.throttle import AutoThrottle
+from twisted.internet import defer
+import twisted.trial.unittest
```

Comment:
as per pep8, twisted imports should go in a separate group, between stdlib imports and scrapy imports